lib/eye/notify/awssdk.rb

```diff
@@ -0,0 +1,43 @@
+require "aws-sdk-core"
+require "aws-sdk-core/ses"
+
+module Eye
+  module Notify
+    class AWSSDK < Eye::Notify
+      param :region, String, true
+      param :access_key_id, String, true
+      param :secret_access_key, String, true
+      param :from, String, true
+
+      def execute
+        client = Aws::SES::Client.new(
+          region: region,
+          credentials: Aws::Credentials.new(access_key_id, secret_access_key))
```

Comment:
Credentials are not required if a user is using IAM credentials or environment variables to supply credentials. It would be nice to make this optional.